roles/monitored/vars/main.yml

```diff
@@ -1,12 +1,12 @@
 nrpe_allowed_ips:
   - 127.0.0.1
   - ::1
-  - 10.5.0.243  # Monitor
```

Comment:
??

Reply:
just changed `Monitor 2` -> `monitor2` to make it clear which machine it is, and did the same for monitor